buildlib/pr/ucx-py/ucx-py-test.yml

```diff
@@ -0,0 +1,90 @@
+parameters:
+  name: ucx-py test
+  demands: []
+
+jobs:
+  - job: ${{ parameters.name }}
+
+    pool:
+      name: MLNX
+      demands: ${{ parameters.demands }}
+
+    steps:
+      - checkout: self
+        fetchDepth: 100
+        clean: true
+        displayName: Checkout
+      - bash: |
+          set -x
+          source buildlib/az-helpers.sh
+          az_init_modules
+          try_load_cuda_env
+          set -eE
+          ./autogen.sh
+          ./contrib/configure-devel --prefix=$(Build.Repository.LocalPath)/install \
+            --enable-gtest=no --with-cuda=$have_cuda
+          make -j`nproc`
+          make install
+        displayName: Build UCX
+      - bash: |
+          set -x
+          source buildlib/az-helpers.sh
+          az_init_modules
+          try_load_cuda_env
+          set -eE
+
+          ifaces=`get_rdma_interfaces`
+          if [ -z "$ifaces" ]; then
+              azure_log_warning "No active RDMA interfaces on machine"
+              exit 0;
+          fi
+
+          # Set path and build parallel level
+          export PATH=/opt/conda/bin:$PATH
+
+          ################################################################################
+          # SETUP - Install conda packages and check environment
+          ################################################################################
+          gpuci_logger "Check environment"
+          env
+
+          gpuci_logger "Activate conda env"
+          . /opt/conda/etc/profile.d/conda.sh
+          conda activate ucx-py
+          gpuci_conda_retry install \
+              "pytest" "pytest-asyncio" \
+              "dask" "distributed" \
+              "cupy=9" "numba" \
+              "cython" "libhwloc"
+
+          gpuci_logger "Clone and Build UCX-Py"
+          git clone https://github.com/rapidsai/ucx-py
+          cd ucx-py
+          python setup.py build_ext --inplace
+          python -m pip install -e .
+
+          gpuci_logger "Check versions"
+          python --version
+          conda info
+          conda config --show-sources
+          conda list --show-channel-urls
+          ucx_info -v
+
+          ################################################################################
+          # TEST - Run UCX-Py tests and benchmarks
+          ################################################################################
+          for tls in "tcp,cuda_copy" "rc,tcp,cuda_copy" "all"; do
+              gpuci_logger "Check GPU usage"
+              nvidia-smi
+
+              # Test with TCP/Sockets
+              gpuci_logger "Tests (UCX_TLS=$tls)"
+              pytest --cache-clear -vs ucp/_libs/tests
+              pytest --cache-clear -vs tests/
+
+              gpuci_logger "Benchmarks (UCX_TLS=$tls)"
```

Comment:
where do we actually set `UCX_TLS=$tls`?

Reply:
Good catch, I must have accidentally removed it when translating the bash script to the yaml file, fixed in https://github.com/openucx/ucx/pull/7412/commits/b7c6c2077b4fde6b95102be4475a6f5774bd2783 .